audits/internal3/README.md

```diff
@@ -0,0 +1,81 @@
+# meme-ooorr
+The review has been performed based on the contract code in the following repository:<br>
+`https://github.com/dvilelaf/meme-ooorr` <br>
+commit: `9b427618db7bd233651476de9abe12c18fbd236a` or `v0.2.0-internal-audit2` <br>
+
+## Objectives
+The audit focused on Meme* contracts <BR>
+
+### Flatten version
+Flatten version of contracts. [contracts](https://github.com/dvilelaf/meme-ooorr/blob/main/audits/internal3/analysis/contracts)
+
+### Security issues
+Details in [slither_full](https://github.com/dvilelaf/meme-ooorr/blob/main/audits/internal3/analysis/slither_full.txt) <br>
+All false positive.
+
+## Issues
+### Medium? Critical?: to discussion, sandwich attack
+```
+attaсker contract => function unleashThisMeme => _createThisMeme => 
+bytes32 randomNonce = keccak256(abi.encodePacked(block.timestamp, block.prevrandao, msg.sender, memeNonce));
+        randomNonce = keccak256(abi.encodePacked(randomNonce));
+Can be calculated in the attacking contract (because deterministic in same block)
```

Comment:
Whilst in theory possible, practically you can always increase your gas payment to get into the block a the start. So there's a straightforward way to circumvent.